### ISSUE TITLE
Logging Configuration

### ISSUE THREAD

**Issue by kentoj** [loftuxab/alfresco-ubuntu-install#69]
It might be nice to have a wiki entry on logging and where the default log files are so that I can easily manage the size and aggregation of the log files.

Is the wiki page a good idea?

**Comment by chris001** [comment]
Yes Alfresco puts log files in many places on the filesystem and so a wiki entry about where the logs are, what each one of them is for, would be great.

**Comment by housni** [comment]
From what I can recall, if you start Alfresco via your init system (upstart or systemd, for example), the logs would be in `/opt/alfresco/logs` as well as somewhere in `/opt/alfresco/tomcat/`. The latter are the Tomcat logs, of course. The former is for Solr, Share and Alfresco.

If, however, you ran the startup shell scripts yourself, in the terminal, the logs will be created at the location you ran the startup script.

**Comment by loftux** [comment]
In 14.04 when using upstart the alfresco log is in /etc/logs/upstart/alfresco/
You will also find tomcat logs in /opt/alfresco/logs, this lines 38-41 makes sure the logs are collected there
https://github.com/loftuxab/alfresco-ubuntu-install/blob/master/tomcat/alfresco.conf#L38

Will probably have to change for 16.04 since it will use Systemd.

**Comment by douglascrp** [comment]
@loftux As there is no more support for 14.04, I believe this one is no more relevant, right?
May we close it?
